common/changes/@rushstack/heft-storybook-plugin/bartvandenende-wm-storybook-improvements_2024-01-03-11-53.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft-storybook-plugin",
+      "comment": "Add support for storybook 7, HMR and storybookPackageNameTarget",
```

```suggestion
      "comment": "Add support for storybook 7, HMR, and storybookPackageNameTarget",
```

Can you also give some more information about what storybookPackageNameTarget is in this changelog message?